### ISSUE TITLE
Staging Review finding: Warning alert icon is not exposed to assistive technology

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128350]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments Team
**Product:** Appointments
**Feature:** Oracle Health Scheduling - Direct Schedule and Request for an Appointment

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On the Appointments screen, the warning alert does not include a text alternative for the icon, or screen reader only text, to indicate what type of alert it is. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="409" height="246" alt="warning icon" src="https://github.com/user-attachments/assets/c97d5003-d130-4359-9c78-d89afb693219" />

</details>

## Recommended action

It appears that this alert implementation is the Mobile app version, and the standard warning alert should be used, which will resolve this issue. 

https://design.va.gov/storybook/?path=/story/uswds-va-alert--warning

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.4.1 Use of Colour - Level A](https://www.w3.org/TR/WCAG22/#use-of-color)
- **Modality:** Screen reader
- **Design System Component:** [Alert component](https://design.va.gov/components/alert)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@kristenmcconnell @sterkenburgsara I don't think we need to do this one since these alerts are going away next month, right?

Calling this out too in case it's a problem with the new alerts (likely not)

**Comment by outerpress** [comment]
Thanks @jasonday ! We'll recontextualize this as fixing that issue across all the alerts. FYA @ala-yna @JunTaoLuo we might need to break this up
